src/components/Table/GridTable.tsx

```diff
@@ -337,11 +330,8 @@ export function GridTable<R extends Kinded, X extends Only<GridTableXss, X> = an
   }, [as, api, style, rowStyles, maybeStyle, columnSizes, getCount, filterMaxRows]);
 
   // Push back to the caller a way to ask us where a row is.
-  const { rowLookup } = props;
-  if (rowLookup) {
-    // Refs are cheap to assign to, so we don't bother doing this in a useEffect
-    rowLookup.current = createRowLookup(columns, visibleDataRows, virtuosoRef);
```

Comment:
`RowLookup` was the last place using the `[Row, ReactElement]` tuples, so I moved it over to use the `GridTableApi` internally...

Really the `RowLookup` API should be removed, as it's been superceded by `GridTableApi`, but am procrastinating on that one.